protocols/5f7d18/5f7d18.ot2.apiv2.py

```diff
@@ -0,0 +1,205 @@
+    mastermix_vol_case2 = mix_vol_per_col*(num_col-6)
+    case2_str = f"and {mastermix_vol_case2}ul in tube A2"if num_col > 6 else ""
+
+    ctx.pause(f'''Please confirm that the following volumes are in their
+respective labware before beginning the protocol:\n
+Minimum {lys_vol_case1}ul of MagBio CTL medium in row A1 {case2_str}
+of the Nest 15mL reservoir on Slot 11.\n
+Minimum {isoprop_beads_vol}ul of isopropanol + magnetic bead mastermix is in
+each row from row A4 to row A7 of the nest 15mL reservoir on Slot 11.\n
+Minimum {nuc_free_water_vol}ul of nuclease free water in row 10 of the
+Nest 15mL reservoir on Slot 11.\n
+Minimum {ethanol_vol}ul of ethanol for each row up to row {num_col} of the
+Nest 15mL reservoir on Slot 10.\n
+Minimum {mastermix_vol_case1}ul of PCR mastermix in tube A1 {case2_str}
+of the Nest 15mL reservoir on Slot 11.\n''')
+
+    # add MagBio CTL medium lysis buffer
+    ctx.comment('\n--------- ADDING MAGBIO CTL MEDIUM ---------\n')
+    for i, col in enumerate(mag_plate.rows()[0][:num_col]):
+        pick_up(m300)
+        m300.transfer(290,
+                      lysis_buffer[0 if i < 6 else 1],
+                      col.top(),
+                      new_tip='never')
+        m300.mix(samp_and_lys_rep, 200, col)
+        m300.drop_tip()
+
+    # add binding buffer
+    ctx.comment('\n--------- ADDING BINDING BUFFER ---------\n')
+    pick_up(m300)
+    for s, col in zip(isoprop_beads_ext, mag_plate.rows()[0][:num_col]):
+        m300.transfer(510,
+                      s,
+                      col.top(),
+                      new_tip='never')
+    m300.drop_tip()
+    for col in mag_plate.rows()[0][:num_col]:
+        pick_up(m300)
+        m300.mix(15, 200, col)
+        m300.drop_tip()
+
+    ctx.delay(minutes=incubate_bind_time)
+
+    # resuspend MAG-S1 particles before engaging magnet
+    ctx.comment('\n--------- RESUSPEND MAG-S1 BEADS BEFORE ENGAGE ---------\n')
+    for col in mag_plate.rows()[0][:num_col]:
+        pick_up(m300)
+        m300.mix(shake_well, 200, col)
+        m300.drop_tip()
+
+    # engage magnetic module, remove supernatant with 2 ethanol washes
+    ctx.comment('\n--------- ENGAGE MAGDECK WITH 2 ETHANOL WASHES ---------\n')
+    for i in range(3):
+        mag_deck.engage(height_from_base=mag_engage_height)
+        ctx.delay(minutes=mag_engage_time)
+
+        # remove supernat
+        ctx.comment('\n--------- REMOVING SUPERNATANT ---------\n')
+        for index, (s_col, d_col) in enumerate(zip(
+             mag_plate.rows()[0][:num_col], supernat.rows()[0])):
+            side = -1 if index % 2 == 0 else 1
+            aspirate_loc = s_col.bottom(asp_height).move(
+                    Point(x=(s_col.diameter/2-length_from_side)*side))
+            pick_up(m300)
+            m300.transfer(500 if i > 0 else 1000,
+                          aspirate_loc,
+                          d_col,
+                          new_tip='never')
+            m300.drop_tip()
```

Comment:
I would change the dispense height for you supernatant (**d_col**) so that the tips are never touching the supernatant